test: fix warnings from scan-build static analyzer

scan-build warnings from this code:

```
../test/integration/session-util.c:192:24: warning: The left operand of '==' is a garbage value due to array index out of bounds
        if (handles[i] == TPM2_RH_NULL)
            ~~~~~~~~~~ ^
../test/integration/session-util.c:238:24: warning: The left operand of '==' is a garbage value due to array index out of bounds
        if (handles[i] == TPM2_RH_NULL)
            ~~~~~~~~~~ ^
2 warnings generated.
```

I will work on this.

--- a/test/integration/session-util.c
+++ b/test/integration/session-util.c
@@ -24,6 +24,8 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <inttypes.h>
+
 #include "session-util.h"
 #include "sapi-util.h"
 #include "context-util.h"
@@ -187,8 +189,14 @@ TSS2_RC ComputeCommandHmacs(
     TPM2B_MAX_BUFFER hmac_key;
     TSS2_RC rval = TPM2_RC_SUCCESS;
     unsigned int i;
+    unsigned int count = pSessionsDataIn->count;
 
-    for (i = 0; i < pSessionsDataIn->count; i++) {
+    if (count > 3) {
+        LOG_ERROR("Bad value for session count: %" PRIu16, count);
+        return TSS2_SYS_RC_GENERAL_FAILURE;
+    }
+
+    for (i = 0; i < count; i++) {
         if (handles[i] == TPM2_RH_NULL)
             break;
 
@@ -233,8 +241,14 @@ TSS2_RC CheckResponseHMACs(
     TPM2B_MAX_BUFFER hmac_key;
     TSS2_RC rval = TPM2_RC_SUCCESS;
     unsigned int i;
+    unsigned int count = pSessionsDataIn->count;
 
-    for (i = 0; i < pSessionsDataIn->count; i++) {
+    if (count > 3) {
+        LOG_ERROR("Bad value for session count: %" PRIu16, count);
+        return TSS2_SYS_RC_GENERAL_FAILURE;
+    }
+
+    for (i = 0; i < count; i++) {
         if (handles[i] == TPM2_RH_NULL)
             break;
 

